javascript-modal/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Javascript Modal</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+  </head>
+  <body>
+    <div>
+      <button class="open-button">Open Modal</button>
+    </div>
+    <div class="modal">
+      <fieldset>
```

Comment:
fieldset and label are used within a form element. I would remove these and use standard text elements if you are not using a form.

Reply:
Thanks for the comment. Updated html/css/js files accordingly. Please review, thanks.
![image](https://user-images.githubusercontent.com/76827487/115598242-0bfbb380-a28f-11eb-9f09-317da425ec96.png)